yarn-project/circuits.js/src/structs/contract_storage_update_request.ts

```diff
@@ -53,11 +53,11 @@ export class ContractStorageUpdateRequest {
     return [fields.storageSlot, fields.newValue, fields.sideEffectCounter] as const;
   }
 
-  static empty() {
+  static default() {
     return new ContractStorageUpdateRequest(Fr.ZERO, Fr.ZERO);
   }
 
-  isEmpty() {
+  isDefault() {
```

Comment:
Again, we are not checking all the values here, so some of them might be "non-standard" but still pass the default?